docs/source/config_files.rst

```diff
@@ -240,10 +240,13 @@ In the majority of cases, providing your own Dockerfile is not necessary as the
 images provide core functionality, compact image sizes, and efficient builds. We recommend
 trying the other configuration files before deciding to use your own Dockerfile.
 
-With Dockerfiles, a regular Docker build will be performed.
+Try the ``--appendix`` :ref:`CLI flag <usage-cli>` as a last resort. If you
+still can't achieve your goals, you should create a Dockerfile from scratch to
+define your image.
 
 .. note::
-    If a Dockerfile is present, all other configuration files will be ignored.
+   With Dockerfiles, a regular Docker build will be performed. All other
+   configuration files will be ignored.
```

Comment:
@manics I'd like to add a sentence here clarifying what value `repo2docker` is providing to the user in this case. e.g. `All other configuration files will be ignored, but \`repo2docker\` will still ___`? Thoughts?

Thanks for your input on this change :)